src/containers/Layout.js

```diff
@@ -1,20 +1,20 @@
-const Connect = require('react-redux').connect;
+const { useSelector } = require('react-redux');
+const { useMiddleEnd } = require('strange-middle-end');
 const Layout = require('../components/Layout');
-const M = require('../middle-end');
 
 const internals = {};
 
-internals.connect = Connect(
-    (state) => ({
-        isAuthenticated: M.selectors.auth.getIsAuthenticated(state)
-    }),
-    (dispatch) => ({
-        logout: async () => {
+module.exports = function LayoutContainer(props) {
 
-            const [err, result] = await M.dispatch.auth.logout();
-            return [err, result];
-        }
-    })
-);
+    const M = useMiddleEnd();
```

Comment:
The original upper-case `M` came from the fact that it was `require()`'d.
```suggestion
    const m = useMiddleEnd();
```